salt/utils/atomicfile.py

```diff
@@ -120,6 +121,8 @@ def close(self):
         self._fh.close()
         if os.path.isfile(self._filename):
             shutil.copymode(self._filename, self._tmp_filename)
+            st = os.stat(self._filename)
+            os.chown(self._tmp_filename, st[stat.ST_UID], st[stat.ST_GID])
```

Comment:
Is there a reason why thisisbeing done using indexing instead of using `st.st_uid` and `st.st_gid`? Is there platform compatibility issue that using the `stat` module attributes helps to disambiguate? It seems to me that doing an attribute lookup would be quicker than doing an attribute lookup _and_ an index.